File: Backend/Models/Duraterelease.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace Relmonitor.Models
{
    public partial class Duraterelease
    {
        public Duraterelease()
        {
            Releases = new HashSet<Release>();
        }

        public int Iddurata { get; set; }
        public string Luna { get; set; }
        public string Saptamana { get; set; }
        public DateTime Datarelease { get; set; }
        public DateTime Durata { get; set; }
        public DateTime Downtime { get; set; }
        public DateTime Datastart { get; set; }
        public DateTime Dataend { get; set; }
        public DateTime Datacreare { get; set; }
        public DateTime Datamodificare { get; set; }

        public virtual ICollection<Release> Releases { get; set; }
    }
}

File: Frontend/src/app/models/duraterelease.ts
import { Release } from "./release";

export class Duraterelease {
    iddurata: number = 0;
    luna: string = '';
    saptamana: string = '';
    datarelease: string = '';
    durata: string = '';
    downtime: string = '';
    datastart:string = '';
    dataend:  string = '';
    datacreare: Date = new Date();
    datamodificare: Date = new Date();
    releases: Release[] = [];
}
